ui/toast/analysis.js

```diff
@@ -0,0 +1,133 @@
+const Color = require("../color");
+
+/**
+ * @class UI.Toast
+ * @since 4.4.1
+ * Toast component allows users to show messages to user for period of time.
```

Comment:
snackbar and toast terms leads to confusion. so need to use one of them. that applies to other places as well (defination and code)